examples/cdi-qualifier/README_pt.adoc

```diff
@@ -0,0 +1,99 @@
+= CDI @Qualifier
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+== Introdução
+
+As vezes nós need escrever várias implementações para uma interface de regras de negócios, para aumentar o desacoplamento vamos injetar apenas a interface e o CDI vai escolher a implementação correta, para ajudar o CDI com esta escolha nós criamos os **qualificadores**.
+
+== Exemplo
+
+Neste exemplo, temos uma interface `Payment` e suas implementações: 
+* Cash
+* CreditCard 
+
+Em nosso teste (Payment Test) nós injetamos somente a interface Payment, sem o recurso Qualificador, o CDI não saberia qual implementação injetar no teste.
+
+Nós criamos um **qualificador** chamado `PaymentQualifier` um único qualificador com uma diferença, a anotação `@Qualifier`.
+
+....
+@Retention(RUNTIME)
+@Target({ TYPE, FIELD, METHOD })
+@Qualifier
+public @interface PaymentQualifier {
+
+	PaymentType type();	
+}
+....
+
+Este qualificador tem um método chamado `type()`, tEsse método ajudará o CDI a injetar corretamente a implementação. veja este enum:
```

Comment:
in the sentence: "métódo chamado type(), 'tEsse' método". The word 'esse' don't have "t"

Reply:
done